packages/react/src/components/cookieConsent/page/Page.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+
+import { Content, Provider as CookieContextProvider } from '../CookieConsentContext';
+import Details from '../details/Details';
+import styles from '../CookieConsent.module.scss';
+import Buttons from '../buttons/Buttons';
+import { Notification } from '../../notification/index';
+
+export function Page(props: { content: Content; cookieDomain?: string }): React.ReactElement | null {
+  const { cookieDomain, content } = props;
+  const { title, text } = content.texts.sections.main;
+  const { settingsSaved } = content.texts.ui;
+  const [showSaveNotification, setShowSaveNotification] = useState(false);
+  const onButtonClick = () => {
+    setShowSaveNotification(true);
+  };
+  return (
+    <CookieContextProvider content={content} cookieDomain={cookieDomain}>
+      <div className={styles.page} data-testid="cookie-consent">
+        <div className={styles.content} id="cookie-consent-content">
+          <div className={styles['main-content']} data-testid="cookie-consent-information">
+            <h1>{title}</h1>
+            <p>{text}</p>
+          </div>
+          <Details />
+          {showSaveNotification && (
```

Comment:
Think we need to tweak the styles of the save notification:

<img width="1135" alt="image" src="https://user-images.githubusercontent.com/2777633/165997668-ddf5ff29-1338-49f0-b70a-0fe1b5e4bedb.png">

Maybe we need to add some spacing between the buttons and the notification? What do you think @ronijaakkola @NikoHelle